### PR TITLE
feat: adapting the error message when request fails because of a 503

### PR DESCRIPTION
**Pull request by pedroferreira1**
Screenshot 2019-04-26 at 04 09 47

### CODE REVIEW

File: src/api/axiosInstance.js [hunk 1]
```diff
@@ -41,6 +41,13 @@ const createRequestInstance = (resolve, timeout) => {
   instance.interceptors.response.use((response) => {
     return response;
   }, (error) => {
+    // Update error message in redux depending on the status code
+    const statusCode = error.response.status;
+    if (statusCode === 503) {
+      store.dispatch(updateRequestErrorMessage('Our server has reached its requests limit. You should wait a few seconds and try again.'));
```
Comment (msbrogli) [inline src/api/axiosInstance.js]: I prefer: "Rate limit exceeded. Sorry about that. You should wait a few seconds and try again."
What do you think?

File: src/components/RequestError.js [hunk 2]
```diff
@@ -85,7 +88,7 @@ class RequestErrorModal extends React.Component {
               </button>
             </div>
             <div className="modal-body">
-              <p>Your request failed to reach the server. What do you want to do?</p>
+              <p>{this.props.requestErrorMessage} What do you want to do?</p>
```
Comment (msbrogli) [inline src/components/RequestError.js]: Even though "What do you want to do?" appears in both cases, I think we should move it to the `this.props.requestErrorMessage` (and add them to the end of the messages).

File: src/api/axiosInstance.js (hunk 1 above)
Comment (pedroferreira1) [reply src/api/axiosInstance.js]: I think it's better too, changed it!

File: src/components/RequestError.js (hunk 2 above)
Comment (pedroferreira1) [reply src/components/RequestError.js]: Makes sense, so in the future we can have messages without this part

**Pull request merged**